### PR TITLE
Initial commit for helm chart

### PR DESCRIPTION
**Pull request by jiangphcn**
## Overview

Build Kepler Helm Chart. 

## Testing recommendations

Run the following command after `make cluster-up`.
```
[root@brooks1 kepler-helm-chart]# /var/lib/snapd/snap/bin/helm install  kepler . --values values.yaml  --create-namespace  --namespace kepler
NAME: kepler
LAST DEPLOYED: Sun Nov 27 05:05:41 2022
NAMESPACE: kepler
STATUS: deployed
REVISION: 1
NOTES:
1. Get the application URL by running these commands:
  export POD_NAME=$(kubectl get pods --namespace kepler -l "app.kubernetes.io/name=kepler-exporter,app.kubernetes.io/instance=kepler" -o jsonpath="{.items[0].metadata.name}")
  export CONTAINER_PORT=$(kubectl get pod --namespace kepler $POD_NAME -o jsonpath="{.spec.containers[0].ports[0].containerPort}")
  echo "Visit http://127.0.0.1:8080 to use your application"
  kubectl --namespace kepler port-forward $POD_NAME 8080:$CONTAINER_PORT
[root@brooks1 kepler-helm-chart]# kubectl get service -n kepler
NAME              TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)    AGE
kepler-exporter   ClusterIP   None         <none>        9102/TCP   29s
[root@brooks1 kepler-helm-chart]# kubectl get DaemonSet -n kepler
NAME              DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
kepler-exporter   1         1         1       1            1           <none>          41s
[root@brooks1 kepler-helm-chart]# kubectl get pod -n kepler
NAME                    READY   STATUS    RESTARTS   AGE
kepler-exporter-znbhv   1/1     Running   0          48s
```

## Checklist

- [X] Code is written and works correctly
- [X] Changes are covered by tests

### CODE REVIEW

**Comment by rootfs** [comment]
@sallyom

File: templates/tests/test-connection.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  labels:
+    {{- include "kepler.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "kepler.name" . }}:{{ .Values.service.port }}']
```
Comment (SamYuan1990) [inline templates/tests/test-connection.yaml:14]: duplicate with pod healthz?

File: Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name:  kepler
+description: A Helm chart for kepler
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```
Comment (SamYuan1990) [inline Chart.yaml:6]: `1.16.0` is version for helm?

**Comment by SamYuan1990** [comment]
It's great to see we have 1st version of helm chart for kepler.

File: Chart.yaml (hunk 2 above)
Comment (jiangphcn) [reply Chart.yaml:6]: it is version generated by helm

Comment (jiangphcn) [reply Chart.yaml:6]: [templates/tests/test-connection.yaml](https://github.com/sustainable-computing-io/kepler-helm-chart/pull/1/files/0e4fd75bfef3e72ed8c824783a09178bb2886cb9#diff-e717f8763feaf34081f7215ef0388559cf3b5fdb60db18750e2ebcaabb1d4af8) was replaced with .github/workflows/integration_test.yml

**Comment by jiangphcn** [comment]
@sallyom and @SamYuan1990  I have added CI and could you please check again?

**Comment by rootfs** [comment]
@jiangphcn thank you! can you add a instruction page [here](https://github.com/sustainable-computing-io/kepler-doc/tree/main/docs/installation)?

**Comment by jiangphcn** [comment]
thanks @rootfs for merging to main. Yes, I will add instruction to [here](https://github.com/sustainable-computing-io/kepler-doc/tree/main/docs/installation) tomorrow.

**Comment by rootfs** [comment]
sounds great @jiangphcn

**Comment by jiangphcn** [comment]
@rootfs  https://github.com/sustainable-computing-io/kepler-doc/pull/35 was issued and ready for review.

**Pull request merged**